artifact_requirements.txt

```diff
@@ -14,4 +14,6 @@ tables
 vivarium_inputs[data]>=5.0.0
 vivarium_cluster_tools>=2.0.0
 black==22.3.0
-isort
\ No newline at end of file
+isort
+jupyterlab
+matplotlib
```

Comment:
we almost certainly don't need matplotlib for artifacts, right? Or is it something RT specifically asked for?

Reply:
These are libraries for RT.